misc/config/config.yaml.nydus.tmpl

```diff
@@ -51,6 +51,8 @@ converter:
       merge_manifest: false
       # nydus chunk dict image reference, used for chunk-leveled data deduplication.
       # chunk_dict_ref: localhost/chunk_dict/image:latest
+      # flatten all source image layers to a nydus layer (one blob + one bootstrap).
```

Comment:
It'd be better to put it "flatten all source image layers to a single nydus layer"

Reply:
Fixed.